.vscode/settings.json

```diff
@@ -36,5 +36,6 @@
   "powershell.codeFormatting.whitespaceAfterSeparator": true,
 
   // Omnisharp : Enable EditorConfig support
```

Comment:
This file should not be part of the PR.

Reply:
ah right, I think my editor added that. Let me resolve that :)